packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
+const GLOB_PATTERNS = '**\/*.{html,js,css}';
```

Comment:
Cool, could you open an issue with what you think is the best set of defaults - just so everyone can weigh in, then remove this and handle errors in the plugin if the developer doesn't define them?

It will suck for a bit, but should get fixed outside of the plugin soon :)